src/js/model/schedule.js

```diff
@@ -269,21 +269,9 @@ Schedule.prototype.init = function(options) {
 };
 
 Schedule.prototype.setAllDayPeriod = function(start, end) {
-    // If it is an all-day schedule, only the date information of the string is used.
-    if (util.isString(start)) {
-        start = datetime.parse(start.substring(0, 10));
-    } else {
-        start = new TZDate(start || Date.now());
-    }
-    if (util.isString(end)) {
-        end = datetime.parse(end.substring(0, 10));
-    } else {
-        end = new TZDate(end || this.start);
-    }
-
-    this.start = start;
+    this.start = new TZDate(start || Date.now());
     this.start.setHours(0, 0, 0);
-    this.end = end || new TZDate(this.start);
+    this.end = new TZDate(end || this.start);
     this.end.setHours(23, 59, 59);
```

Comment:
You can use `start`, `end` of datetime
https://github.com/nhn/tui.calendar/blob/master/src/js/common/datetime.js#L396
https://github.com/nhn/tui.calendar/blob/master/src/js/common/datetime.js#L408
```js
    this.start = datetime.start(new TZDate(start || Date.now()));
    this.end = datetime.end(new TZDate(end || this.start));
```

Reply:
just modified.